INFO 2025-06-24 16:54:47,891 autoreload Watching for file changes with StatReloader
INFO 2025-06-24 16:54:57,581 basehttp "GET /documents/ HTTP/1.1" 200 17375
INFO 2025-06-24 16:55:01,583 basehttp "POST /documents/1/process/ HTTP/1.1" 200 123
INFO 2025-06-24 16:55:16,326 autoreload Watching for file changes with StatReloader
INFO 2025-06-24 16:56:05,467 autoreload /home/akemi/Bureau/Gnosis/last/django-manifest-app/manifest_project/settings.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO Watching for file changes with StatReloader
INFO /home/akemi/Bureau/Gnosis/last/django-manifest-app/manifest_project/settings.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO "GET /documents/ HTTP/1.1" 200 17375
INFO /home/akemi/Bureau/Gnosis/last/django-manifest-app/manifest_project/settings.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO "POST /documents/1/process/ HTTP/1.1" 200 191
INFO "GET /documents/ HTTP/1.1" 200 17717
ERROR Internal Server Error: /documents/1/results/
Traceback (most recent call last):
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/contrib/auth/decorators.py", line 60, in _view_wrapper
    return view_func(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/Bureau/Gnosis/last/django-manifest-app/manifest_app/views.py", line 274, in pdf_results
    return render(request, 'manifest_app/pdf_results.html', context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/template/loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/template/backends/django.py", line 79, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/template/engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/template/engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/template/loaders/cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/template/loaders/base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/template/base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/template/base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/template/base.py", line 518, in parse
    raise self.error(token, e)
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/template/base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/template/loader_tags.py", line 295, in do_extends
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/template/base.py", line 518, in parse
    raise self.error(token, e)
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/template/base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/template/loader_tags.py", line 234, in do_block
    nodelist = parser.parse(("endblock",))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/template/base.py", line 518, in parse
    raise self.error(token, e)
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/template/base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/template/defaulttags.py", line 962, in do_if
    nodelist = parser.parse(("elif", "else", "endif"))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/template/base.py", line 489, in parse
    raise self.error(token, e)
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/template/base.py", line 487, in parse
    filter_expression = self.compile_filter(token.contents)
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/template/base.py", line 605, in compile_filter
    return FilterExpression(token, self)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/template/base.py", line 678, in __init__
    raise TemplateSyntaxError(
django.template.exceptions.TemplateSyntaxError: Could not parse some characters: manifest_entries.values|:'vessel'||length
ERROR "GET /documents/1/results/ HTTP/1.1" 500 255067
INFO "GET /donnees/ HTTP/1.1" 200 14613
INFO "GET /donnees/ HTTP/1.1" 200 14613
INFO "GET /documents/ HTTP/1.1" 200 17717
INFO "GET /documents/upload/ HTTP/1.1" 200 15889
INFO "POST /documents/upload/ HTTP/1.1" 302 0
INFO "GET /documents/ HTTP/1.1" 200 20542
INFO /home/akemi/Bureau/Gnosis/last/django-manifest-app/manifest_app/services/ai_service.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO "GET /documents/ HTTP/1.1" 200 20191
INFO "POST /documents/2/process/ HTTP/1.1" 200 191
INFO "GET /documents/ HTTP/1.1" 200 20533
ERROR Internal Server Error: /documents/2/results/
Traceback (most recent call last):
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/contrib/auth/decorators.py", line 60, in _view_wrapper
    return view_func(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/Bureau/Gnosis/last/django-manifest-app/manifest_app/views.py", line 274, in pdf_results
    return render(request, 'manifest_app/pdf_results.html', context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/template/loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/template/backends/django.py", line 79, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/template/engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/template/engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/template/loaders/cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/template/loaders/base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/template/base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/template/base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/template/base.py", line 518, in parse
    raise self.error(token, e)
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/template/base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/template/loader_tags.py", line 295, in do_extends
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/template/base.py", line 518, in parse
    raise self.error(token, e)
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/template/base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/template/loader_tags.py", line 234, in do_block
    nodelist = parser.parse(("endblock",))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/template/base.py", line 518, in parse
    raise self.error(token, e)
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/template/base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/template/defaulttags.py", line 962, in do_if
    nodelist = parser.parse(("elif", "else", "endif"))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/template/base.py", line 489, in parse
    raise self.error(token, e)
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/template/base.py", line 487, in parse
    filter_expression = self.compile_filter(token.contents)
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/template/base.py", line 605, in compile_filter
    return FilterExpression(token, self)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/template/base.py", line 678, in __init__
    raise TemplateSyntaxError(
django.template.exceptions.TemplateSyntaxError: Could not parse some characters: manifest_entries.values|:'vessel'||length
ERROR "GET /documents/2/results/ HTTP/1.1" 500 253895
INFO "GET /donnees/ HTTP/1.1" 200 17325
INFO "GET /utilisateurs/ HTTP/1.1" 200 15353
INFO "GET /documents/ HTTP/1.1" 200 20533
ERROR Internal Server Error: /documents/2/results/
Traceback (most recent call last):
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/contrib/auth/decorators.py", line 60, in _view_wrapper
    return view_func(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/Bureau/Gnosis/last/django-manifest-app/manifest_app/views.py", line 274, in pdf_results
    return render(request, 'manifest_app/pdf_results.html', context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/template/loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/template/backends/django.py", line 79, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/template/engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/template/engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/template/loaders/cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/template/loaders/base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/template/base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/template/base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/template/base.py", line 518, in parse
    raise self.error(token, e)
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/template/base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/template/loader_tags.py", line 295, in do_extends
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/template/base.py", line 518, in parse
    raise self.error(token, e)
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/template/base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/template/loader_tags.py", line 234, in do_block
    nodelist = parser.parse(("endblock",))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/template/base.py", line 518, in parse
    raise self.error(token, e)
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/template/base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/template/defaulttags.py", line 962, in do_if
    nodelist = parser.parse(("elif", "else", "endif"))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/template/base.py", line 489, in parse
    raise self.error(token, e)
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/template/base.py", line 487, in parse
    filter_expression = self.compile_filter(token.contents)
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/template/base.py", line 605, in compile_filter
    return FilterExpression(token, self)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/template/base.py", line 678, in __init__
    raise TemplateSyntaxError(
django.template.exceptions.TemplateSyntaxError: Could not parse some characters: manifest_entries.values|:'vessel'||length
ERROR "GET /documents/2/results/ HTTP/1.1" 500 253895
INFO /home/akemi/Bureau/Gnosis/last/django-manifest-app/manifest_app/views.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO "GET /documents/2/results/ HTTP/1.1" 200 20961
INFO "GET /donnees/ HTTP/1.1" 200 17325
INFO "GET /utilisateurs/ HTTP/1.1" 200 15353
INFO "GET /documents/ HTTP/1.1" 200 20533
INFO "GET /statistiques/ HTTP/1.1" 200 20282
INFO "GET / HTTP/1.1" 200 19086
INFO "GET / HTTP/1.1" 200 19086
ERROR Internal Server Error: /vessels/
Traceback (most recent call last):
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/contrib/auth/decorators.py", line 60, in _view_wrapper
    return view_func(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/Bureau/Gnosis/last/django-manifest-app/manifest_app/views.py", line 134, in vessels
    return render(request, 'manifest_app/vessels.html', context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/template/loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: manifest_app/vessels.html
ERROR "GET /vessels/ HTTP/1.1" 500 91426
INFO "GET /donnees/ HTTP/1.1" 200 17325
INFO "GET /documents/ HTTP/1.1" 200 20533
INFO "GET /statistiques/ HTTP/1.1" 200 20282
INFO /home/akemi/Bureau/Gnosis/last/django-manifest-app/manifest_app/views.py changed, reloading.
INFO Watching for file changes with StatReloader
INFO "GET /statistiques/ HTTP/1.1" 200 20282
INFO "GET / HTTP/1.1" 200 33875
INFO "GET / HTTP/1.1" 200 32910
INFO "GET /statistiques/ HTTP/1.1" 200 20282
INFO "GET /donnees/ HTTP/1.1" 200 17325
INFO "GET /utilisateurs/ HTTP/1.1" 200 15353
ERROR Internal Server Error: /vessels/
Traceback (most recent call last):
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/contrib/auth/decorators.py", line 60, in _view_wrapper
    return view_func(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/Bureau/Gnosis/last/django-manifest-app/manifest_app/views.py", line 221, in vessels
    return render(request, 'manifest_app/vessels.html', context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/template/loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: manifest_app/vessels.html
ERROR "GET /vessels/ HTTP/1.1" 500 91439
ERROR Internal Server Error: /consignes/
Traceback (most recent call last):
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/contrib/auth/decorators.py", line 60, in _view_wrapper
    return view_func(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/Bureau/Gnosis/last/django-manifest-app/manifest_app/views.py", line 245, in consignes
    return render(request, 'manifest_app/consignes.html', context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/template/loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: manifest_app/consignes.html
ERROR "GET /consignes/ HTTP/1.1" 500 91462
INFO "GET / HTTP/1.1" 200 33875
INFO "GET /donnees/ HTTP/1.1" 200 17325
INFO "GET /donnees/ HTTP/1.1" 200 13660
INFO "GET /documents/ HTTP/1.1" 200 16868
INFO "GET / HTTP/1.1" 200 30210
INFO "GET /statistiques/ HTTP/1.1" 200 16617
INFO "GET /utilisateurs/ HTTP/1.1" 200 11688
INFO "GET /statistiques/ HTTP/1.1" 200 16617
INFO "GET / HTTP/1.1" 200 30210
INFO "GET / HTTP/1.1" 200 26545
INFO "GET / HTTP/1.1" 200 30210
INFO "GET / HTTP/1.1" 200 30210
INFO "GET / HTTP/1.1" 200 30210
INFO "GET /statistiques/ HTTP/1.1" 200 16617
INFO "GET /documents/ HTTP/1.1" 200 16868
INFO "GET /donnees/ HTTP/1.1" 200 13660
INFO "GET /documents/ HTTP/1.1" 200 16868
INFO "GET /documents/1/results/ HTTP/1.1" 200 11164
INFO "GET /documents/ HTTP/1.1" 200 16868
INFO "GET /documents/upload/ HTTP/1.1" 200 12234
INFO "POST /documents/upload/ HTTP/1.1" 302 0
INFO "GET /documents/ HTTP/1.1" 200 19691
INFO "POST /documents/3/process/ HTTP/1.1" 200 191
INFO "GET /documents/ HTTP/1.1" 200 19683
INFO "GET /documents/3/results/ HTTP/1.1" 200 29926
INFO "GET /documents/ HTTP/1.1" 200 19683
INFO "GET /utilisateurs/ HTTP/1.1" 200 11688
INFO "GET /donnees/ HTTP/1.1" 200 28135
INFO "GET /documents/ HTTP/1.1" 200 19683
INFO "GET /statistiques/ HTTP/1.1" 200 22509
INFO "GET / HTTP/1.1" 200 42137
INFO "GET /statistiques/ HTTP/1.1" 200 22509
INFO "GET /documents/ HTTP/1.1" 200 19683
INFO "GET /donnees/ HTTP/1.1" 200 28135
INFO "GET /utilisateurs/ HTTP/1.1" 200 11688
INFO "GET /documents/ HTTP/1.1" 200 19683
INFO "GET /statistiques/ HTTP/1.1" 200 22509
INFO "GET / HTTP/1.1" 200 42137
INFO "GET /statistiques/ HTTP/1.1" 200 22509
INFO "GET /statistiques/ HTTP/1.1" 200 27158
INFO "GET / HTTP/1.1" 200 46786
INFO "GET / HTTP/1.1" 200 40482
INFO "GET / HTTP/1.1" 200 41329
INFO "GET / HTTP/1.1" 200 41329
INFO "GET /donnees/ HTTP/1.1" 200 33631
INFO "GET /utilisateurs/ HTTP/1.1" 200 17184
INFO "GET /donnees/ HTTP/1.1" 200 33631
INFO "GET /documents/ HTTP/1.1" 200 25179
INFO "GET /donnees/ HTTP/1.1" 200 33631
INFO "GET /utilisateurs/ HTTP/1.1" 200 17184
INFO "GET /documents/ HTTP/1.1" 200 25179
INFO "GET /statistiques/ HTTP/1.1" 200 28005
INFO "GET / HTTP/1.1" 200 41329
WARNING Not Found: /.well-known/appspecific/com.chrome.devtools.json
WARNING "GET /.well-known/appspecific/com.chrome.devtools.json HTTP/1.1" 404 6478
INFO "GET / HTTP/1.1" 200 41308
INFO "GET / HTTP/1.1" 200 41320
INFO "GET /statistiques/ HTTP/1.1" 200 27996
INFO "GET /documents/ HTTP/1.1" 200 25170
INFO "GET /donnees/ HTTP/1.1" 200 33622
INFO "GET /utilisateurs/ HTTP/1.1" 200 17175
INFO "GET /utilisateurs/ HTTP/1.1" 200 14035
INFO "GET /utilisateurs/ HTTP/1.1" 200 14107
INFO "GET /utilisateurs/ HTTP/1.1" 200 14103
INFO "GET /utilisateurs/ HTTP/1.1" 200 14105
INFO "GET /utilisateurs/ HTTP/1.1" 200 14105
INFO "GET /utilisateurs/ HTTP/1.1" 200 14105
INFO "GET /utilisateurs/ HTTP/1.1" 200 14105
INFO "GET /utilisateurs/ HTTP/1.1" 200 14105
INFO "GET / HTTP/1.1" 200 38250
INFO "GET /statistiques/ HTTP/1.1" 200 24926
INFO "GET /documents/ HTTP/1.1" 200 22100
INFO "GET /statistiques/ HTTP/1.1" 200 24926
INFO "GET /donnees/ HTTP/1.1" 200 30552
INFO "GET /utilisateurs/ HTTP/1.1" 200 14105
INFO "GET /donnees/ HTTP/1.1" 200 30552
INFO "GET / HTTP/1.1" 200 38250
INFO "GET /utilisateurs/ HTTP/1.1" 200 14105
INFO "GET /utilisateurs/ HTTP/1.1" 200 14105
INFO "GET /donnees/ HTTP/1.1" 200 30552
INFO "GET /documents/ HTTP/1.1" 200 22100
INFO "GET / HTTP/1.1" 200 38250
INFO "GET /statistiques/ HTTP/1.1" 200 24926
INFO "GET /statistiques/ HTTP/1.1" 200 24884
INFO "GET / HTTP/1.1" 200 38208
ERROR Internal Server Error: /vessels/
Traceback (most recent call last):
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/contrib/auth/decorators.py", line 60, in _view_wrapper
    return view_func(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/Bureau/Gnosis/last/django-manifest-app/manifest_app/views.py", line 221, in vessels
    return render(request, 'manifest_app/vessels.html', context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akemi/anaconda3/envs/django/lib/python3.12/site-packages/django/template/loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: manifest_app/vessels.html
ERROR "GET /vessels/ HTTP/1.1" 500 91577
INFO "GET /donnees/ HTTP/1.1" 200 30510
INFO "GET /documents/ HTTP/1.1" 200 22058
INFO "GET /statistiques/ HTTP/1.1" 200 24884
